themes/default/content/docs/install/_index.md

```diff
@@ -1,8 +1,8 @@
 ---
 title_tag: "Download & Install Pulumi"
 meta_desc: This page contains detailed instructions for downloading and installing Pulumi.
-title: "Download & install"
-h1: Download & install Pulumi
+title: "Download & Install"
+h1: Download & Install Pulumi
```

Comment:
Any reason why we are adjusting the casing here? I believe we generally do sentence casing as opposed to title casing for our headings unless it is a proper noun. Though it does seem like there are some inconsistencies across the site with regards to this.

Reply:
Gotcha, I wasn't aware of a style around this. Happy to adhere to the sentence casing.